portfolio/src/main/java/com/google/sps/servlets/data/Comment.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets.models;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.List;
+import com.google.gson.Gson;
+
+public class Comment {
+  private long id;
+  private String username;
+  private String content;
+  private long timestamp;
+
+  public Comment(long id, String username, String content, long timestamp) {
+    this.id = id;
+    this.content = content;
+    this.username = username;
+    this.timestamp = timestamp;
+  }
+
+  public static Comment convertToComment(Entity entity) {
```

Comment:
Make this and the below method overloaded constructors.

Reply:
I understand doing this for convertToComment, but why make createNewCommentEntity() an overloaded constructor?